src/wizard/javascript/nextjs_wizard.md

```diff
@@ -1,9 +0,0 @@
-Configure your app automatically with [Sentry wizard](https://docs.sentry.io/platforms/javascript/guides/nextjs/#configure).
-
```

Comment:
@AbhiPrasad this file was causing an issue in the build...I believe we do not need this. can you please confirm?

Reply:
It is being used here: https://github.com/getsentry/sentry-docs/blob/b22f4be33eb20969dc0aecff0705c60b185aa63e/src/wizard/javascript/nextjs.md?plain=1#L6

We still need it.

Reply:
this was removed in this PR https://github.com/getsentry/sentry-docs/blob/d9410a9fdbfba72df3ae9e8c3483d23e999a593c/src/wizard/javascript/nextjs/index.md